frontend/assets/bookmarklets/identity-signup.js

```diff
@@ -0,0 +1,8 @@
+(function() {
+    var username = prompt('Test User key (first name, last name, username and password will be set to this value):');
+    if (!username) return;
+    ['firstName', 'secondName', 'publicFields_username', 'password', 'primaryEmailAddress'].forEach(function(field) {
+        document.getElementById('user_'+field).value = field === 'primaryEmailAddress' ? username + '@@gu.com' : username;
```

Comment:
I need to get rid of this `@@` which was to escape Play's templating - incorrect now that this is in its own js file